<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DialogAddIcepap</class>
 <widget class="QDialog" name="DialogAddIcepap" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Add Icepap System</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="icepapcms.qrc" >:/icons/IcepapCfg Icons/ipapsys.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="8" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Description</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="6" >
    <widget class="QTextEdit" name="txtDescription" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>101</red>
         <green>148</green>
         <blue>235</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>101</red>
         <green>148</green>
         <blue>235</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>84</red>
         <green>123</green>
         <blue>196</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </disabled>
      </palette>
     </property>
    </widget>
   </item>
   <item row="0" column="6" >
    <widget class="QLineEdit" name="txtPort" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>101</red>
         <green>148</green>
         <blue>235</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>101</red>
         <green>148</green>
         <blue>235</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>84</red>
         <green>123</green>
         <blue>196</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </disabled>
      </palette>
     </property>
     <property name="maxLength" >
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLineEdit" name="txtHost" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>101</red>
         <green>148</green>
         <blue>235</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>16</red>
         <green>16</green>
         <blue>16</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>101</red>
         <green>148</green>
         <blue>235</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
        <color>
         <red>119</red>
         <green>117</green>
         <blue>115</blue>
        </color>
        <color>
         <red>159</red>
         <green>157</green>
         <blue>154</blue>
        </color>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>84</red>
         <green>123</green>
         <blue>196</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>247</red>
         <green>245</green>
         <blue>243</blue>
        </color>
       </disabled>
      </palette>
     </property>
    </widget>
   </item>
   <item row="0" column="7" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="icepapcms.qrc" >:/small_icons/IcepapCfg Icons/redpixel.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="icepapcms.qrc" >:/small_icons/IcepapCfg Icons/redpixel.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QLabel" name="label_1" >
     <property name="text" >
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>txtHost</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>txtDescription</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="icepapcms.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogAddIcepap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogAddIcepap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
